getting_started/scripting/gdscript/gdscript_basics.rst

```diff
@@ -392,7 +396,7 @@ accessed as array.
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 2D Rectangle type containing two vectors fields: ``position`` and ``size``.
-Alternatively contains an ``end`` field which is ``position+size``.
+Alternatively contains an ``end`` field which is ``position + size``.
```

Comment:
I'd remove the "Alternatively". Seems to imply the field is optional, but it's always there. "Also" might be better.